pkg/controllers/applicationset_controller.go

```diff
@@ -135,18 +144,19 @@ func (r *ApplicationSetReconciler) createOrUpdateInCluster(ctx context.Context,
 		})
 
 		if err != nil {
-			appLog.WithError(err).Errorf("failed to %s Application", action)
+			appLog.WithError(err).WithField("action", action).Error("failed to create/update Application")
 			continue
```

Comment:
Maybe add a flag to indicate if there were errors, and return the flag at the end of the function

Reply:
Done.